build/copyright/ccri-dstl-copyright-2017.txt

```diff
@@ -1,6 +0,0 @@
-Copyright (c) ${project.inceptionYear}-2016 ${owner}
-Portions Crown Copyright (c) 2017 Dstl
```

Comment:
This doesn't make sense. If dstl is 2017 then why's it 2016 in the new build/copyright/ccri-dstl-copyright-2016.txt ?